railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Allow `bin/rails test` to take absolute paths as arguments
```

Comment:
Skipped this for master and 5-1-stable, but I'll add it specifically for the rcs.